homeassistant/components/binary_sensor/august.py

```diff
@@ -133,6 +139,10 @@ def update(self):
         state_provider = SENSOR_TYPES_DOOR[self._sensor_type][2]
         self._state = state_provider(self._data, self._door)
 
+        from august.lock import LockDoorStatus
+        self._available = False if self._state == LockDoorStatus.UNKNOWN\
```

Comment:
```py
self._available = self._state != LockDoorStatus.UNKNOWN
```

Reply:
Updated locally. Learning every second right now about Python. :-)

Going to wait till balloob has been able to review before doing commit to see if he has any further comments.

Reply:
that's fine 👍